sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ServiceBusProcessorOptions.cs

```diff
@@ -133,19 +138,15 @@ public int MaxConcurrentCalls
         /// <returns>A new copy of <see cref="ServiceBusProcessorOptions" />.</returns>
         internal ServiceBusProcessorOptions Clone()
         {
-            var clone = new ServiceBusProcessorOptions
+            return new ServiceBusProcessorOptions
             {
                 ReceiveMode = ReceiveMode,
                 PrefetchCount = PrefetchCount,
                 AutoComplete = AutoComplete,
                 MaxAutoLockRenewalDuration = MaxAutoLockRenewalDuration,
-                MaxReceiveWaitTime = MaxReceiveWaitTime
+                MaxReceiveWaitTime = MaxReceiveWaitTime,
+                MaxConcurrentCalls = MaxConcurrentCalls
             };
-            if (MaxConcurrentCalls > 0)
```

Comment:
default value calculation was moved into the options classes, so this logic is no longer needed.